### DISCUSSION TITLE
Global shortcut to start application

### DISCUSSION THREAD

**Discussion by francwalter** [autokey/autokey#827] [Q&A] [answered]
Hallo

I dont know how to use Autokey to create a global shortcut for an application.
I have GoldenDict as Appimage from: [Early Access Builds for Linux Portable](https://github.com/goldendict/goldendict/wiki/Early-Access-Builds-for-Linux-Portable) and I want to start it with a shortcut.
It would be on command line:

`/home/f/GoldenDict/GoldenDict-661dd4d-x86_64.AppImage`

That is possible?

Thanks for hints.
frank
Ubuntu 22.04 with Autokey 0.95.10

**Comment by sebastiansam55** [comment, marked as answer]
You'd want to do something like;
```
system.exec_command("/home/f/GoldenDict/GoldenDict-661dd4d-x86_64.AppImage")
```

And then configure the hotkeys as appropriate.

Documentation on that can be found at the link below;
https://autokey.github.io/api/system.html#autokey.scripting.System.exec_command

**Comment by francwalter** [comment]
Thank! This works :)
